opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -259,11 +266,11 @@ message Span {
     // then no attributes were dropped.
     uint32 dropped_attributes_count = 5;
 
-    // Flags, a bit field. 8 least significant bits are the trace
+    // Flags, a bit field. Bit 0-7 (8 least significant bits) are the trace
     // flags as defined in W3C Trace Context specification. Readers
-    // MUST not assume that 24 most significant bits will be zero.
-    // When creating new spans, the most-significant 24-bits MUST be
-    // zero.  To read the 8-bit W3C trace flag (use flags &
+    // MUST not assume that bits 10-31 (22 most significant bits) will be zero.
+    // When creating new spans, bits 10-31 (most-significant 22-bits) MUST be
+    // zero. To read the 8-bit W3C trace flag (use flags &
     // SPAN_FLAGS_TRACE_FLAGS_MASK).  [Optional].
     //
     // See https://www.w3.org/TR/trace-context-2/#trace-flags for the flag definitions.
```

Comment:
@tigrannajaryan Does a comment about the parent is remote flag go here too? I'm not so familiar with the span Link use case.

Reply:
How would the code that creates a Link be able to indicate "has remote parent" information in `Link.flags`? Is that part of the AddLink API?

Reply:
@yurishkuro 
For a Link these flags would indicate whether the linked span is remote (which is strictly speaking not a `remote parent`). The `AddLink` API takes the `SpanContext` (of the span to link to). And the `SpanContext` has the `isRemote` API which indicates whether that SpanContext represents a remote span or not. And this information should be accessible from the exporters, right?

Reply:
I would propose then for clarity that we have a separate enum for use for a Link. Otherwise, we could try to find a term that could refer to a parent and linked span and use one enum, though finding that term might be difficult.
```
enum SpanFlagsLinkedSpanIsRemote {
  // Older clients may not set this field, so 00 represents that this value
  // has not been set.
  SPAN_FLAGS_LINKED_SPAN_IS_REMOTE_UNKNOWN = 0;
  // The linked span is remote: 11
  SPAN_FLAGS_LINKED_SPAN_IS_REMOTE = 0x00000300;
  // The linked span is not remote: 01
  SPAN_FLAGS_LINKED_SPAN_IS_NOT_REMOTE = 0x00000100;
}
```